sources/index.html

```diff
@@ -51,5 +51,179 @@ <h2>I'm a software developer</h2>
+                            <button class="projectview">See Project</button>
+                        </section>
+                    </div>
+                </article>
+                <!--project 4-->
+                <article class="project-list">
+                    <div class="project1">
+                        <img src="../images/ImgPlaholder.png" alt="project-preview">
+                        <h2>Multi-Post Stories<br>
+                        Gain+Glory</h2>
+                        <ul class="button-grid">
+                            <li class="lang-button1">Ruby on rails</li>
+                            <li class="lang-button2">Css</li>
+                            <li class="lang-button3">JavScript</li>
+                            <li class="lang-button4">Html</li>
+                        </ul>
+                        <section class="main-button">
+                            <button class="projectview">See Project</button>
+                        </section>
+                    </div>
+                </article>
+                <!--project 5-->
+                <article class="project-list">
+                    <div class="project1">
+                        <img src="../images/ImgPlaholder.png" alt="project-preview">
+                        <h2>Multi-Post Stories<br>
+                        Gain+Glory</h2>
+                        <ul class="button-grid">
+                            <li class="lang-button1">Ruby on rails</li>
+                            <li class="lang-button2">CSS</li>
+                            <li class="lang-button3">JavaScript</li>
+                            <li class="lang-button4">HTML</li>
+                        </ul>
+                        <section class="main-button">
+                            <button class="projectview">See Project</button>
+                        </section>
+                    </div>
+                </article>
+                <!--Project 6-->
+                <article class="project-list">
+                    <div class="project1">
+                        <img src="../images/ImgPlaholder.png" alt="project-preview">
+                        <h2>Multi-Post Stories<br>
+                        Gain+Glory</h2>
+                        <ul class="button-grid">
+                            <li class="lang-button1">Ruby on rails</li>
+                            <li class="lang-button2">Css</li>
+                            <li class="lang-button3">JavaScript</li>
+                            <li class="lang-button4">HTML</li>
+                        </ul>
+                        <section class="main-button">
+                            <button class="projectview">See Project</button>
+                        </section>
+                    </div>
+                </article>
+            </div>
+        </section>
+        <!--about page-->
+        <section class="about">
+            <div class="aboutme-desc">
```

Comment:
- [ ] Kindly add the matching background to your project
Yours:
![image](https://user-images.githubusercontent.com/73140461/148590363-9c6cf0b1-4fbf-465b-87aa-d7e708c34c13.png)
Design:
![image](https://user-images.githubusercontent.com/73140461/148590417-f1f53995-c8b5-42e7-8d5f-b9f0a3b4f645.png)